### PR TITLE
Getting ready for 7.17.0

### PR DESCRIPTION
**Pull request by robertohuertasm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Change Log
 
+## 7.17.0 (October 29, 2017)
+
+- Fix: Fixed issue with `vscode.nls`. ([@JimiC](https://github.com/JimiC) in [#1319](https://github.com/vscode-icons/vscode-icons/pull/1319))
+- Enhancement: Updated dependencies. ([@JimiC](https://github.com/JimiC) in [#1317](https://github.com/vscode-icons/vscode-icons/pull/1317))
+- Feature: Support for `Apex`. ([@JimiC](https://github.com/JimiC) in [#1316](https://github.com/vscode-icons/vscode-icons/pull/1316))
+- Feature: Support for `Mediawiki`. ([@JimiC](https://github.com/JimiC) in [#1315](https://github.com/vscode-icons/vscode-icons/pull/1315))
+- Feature: Support for `Mojolicious`. ([@JimiC](https://github.com/JimiC) in [#1314](https://github.com/vscode-icons/vscode-icons/pull/1314))
+- Fix: PAD disabled detectigon flow. ([@robertohuertasm](https://github.com/robertohuertasm) in [#1312](https://github.com/vscode-icons/vscode-icons/pull/1312))
```
Comment (JimiC) [inline CHANGELOG.md]: Typo on `detection`,

Comment (JimiC) [reply CHANGELOG.md]: Also, use more descriptive description like `Fixed PAD disabled detection flow.`

Comment (JimiC) [reply CHANGELOG.md]: Any response to the above comment? Did you miss it?

Comment (robertohuertasm) [reply CHANGELOG.md]: Yep, I totally missed it.

**Comment by JimiC** [comment]
Are you gonna draft a new release in `Releases`?

**Comment by robertohuertasm** [comment]
Yes, as usual. Why?

**Comment by JimiC** [comment]
In case you forgot.
Quoting Danny Glover in Lethal Weapon, "I'm too old for this shit..."

**Comment by robertohuertasm** [comment]
You have to consider my speed is limited by family issues. I'm doing all this while trying to get my daughter asleep... :wink:

**Comment by JimiC** [comment]
I'm not judging.

**Pull request merged**